bndtools.release/src/bndtools/release/api/ReleaseContext.java

```diff
@@ -75,13 +77,23 @@ public void setProgressMonitor(IProgressMonitor progressMonitor) {
 
     public void addReleasedJar(Jar jar) {
         releasedJars.add(jar);
+
+        // Save off our summary info now, because our JARs will get
+        // closed out from underneath us later
+        releasedJarSummaries.add(ReleaseUtils.getBundleSymbolicName(jar) + "-" + ReleaseUtils.getBundleVersion(jar));
     }
 
     public List<Jar> getReleasedJars() {
         return releasedJars;
     }
 
+    public List<String> getReleaseSummaries() {
+        return releasedJarSummaries;
+    }
+
     public void close() {
+        // XXX: This code doesn't do anything, these JARs were closed by
+        // ProjectBuilder
```

Comment:
OK, so remove the close loop and leave a comment that indicates the released jars were closed by the project builder.

Reply:
K